pkg/handlers/document/bulkv2_create.go

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2022 Zinc Labs Inc and Contributors
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package document
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/zinclabs/zinc/pkg/core"
+	"github.com/zinclabs/zinc/pkg/ider"
+	"github.com/zinclabs/zinc/pkg/meta"
+)
+
+// Bulkv2 accept JSONIngest json documents. Its a simpler and standard format to ingest data.
+// support use field `_id` set document id
+//
+// @Id Bulkv2
+// @Summary Bulkv2 documents
+// @Tags    Document
+// @Accept  plain
+// @Produce json
+// @Param   index  path  string  true  "Index"
+// @Param   query  body  string  true  "Query"
```

Comment:
`@Accept` should be `json`
`@Param query body string true` should be `@Param query body meta.JSONIngest`